cardano-cli/ChangeLog.md

```diff
@@ -6,6 +6,8 @@
 
 - Allow reading signing keys from a pipe ([PR 4342](https://github.com/input-output-hk/cardano-node/pull/4342))
 
+- Infer protocol params in `transaction build` ([PR 4431](https://github.com/input-output-hk/cardano-node/pull/4431))
```

Comment:
I would change this to "Query protocol parameters from the node in the transaction build command"